docs/SRU/reference/exception-firmware-updates.rst

```diff
@@ -52,25 +48,23 @@ OEMs can add the following to their metadata:
 What can be SRUed
 -----------------
 
-New versions of fwupd and fwupdate can both be SRU'ed into older
-releases provided following process is followed: On an Ubuntu release
-that uses both fwupdate and fwupd (such as bionic and earlier):
+New versions of fwupd and fwupd-efi can both be SRU'ed into older
+releases provided following process is followed:
 
-**fwupdate**: tarball releases only. No backported individual patches.
```

Comment:
> No backported individual patches.

That may be your preference, but it's not the position of the Ubuntu SRU team. Please remove this. I appreciate it was there before; I don't know how that slipped in.

Reply:
>  If Ubuntu is going to stop taking microreleases it's a waste of effort upstream.

I made no suggestion that Ubuntu should stop taking microreleases. In fact I specifically pointed out that our policy permits them. I'm merely refusing to implement a ban on cherry-picking or a recommendation against normal Ubuntu packaging practice. Microrelease updates are still welcome, probably preferable, and can follow the documentation (or its update) as normal.